---
layout: default
title: MIQE
---
<h3>MIQE: Minimum Information for Publication of Quantitative Real-Time PCR Experiments</h3>

<p>The RDML file format is a recommended element in the Minimum Information for Publication of Quantitative
	Real-Time PCR Experiments (MIQE) guidelines
	(<a href="https://pubmed.ncbi.nlm.nih.gov/19246619/" target="_blank">Bustin et al.,
		Clinical Chemistry, 2009</a>).
</p>
<p>
   The aim of MIQE, coordinated by a group of research-active scientists and coordinated under the umbrella of
	<a href="http://www.mibbi.org" target="_blank">MIBBI</a> (Minimum Information for Biological and Biomedical
	Investigations) is to provide authors, reviewers and editors specifications for the minimum information that
	must be reported for a qPCR experiment in order to ensure its relevance, accuracy, correct interpretation and
	repeatability. A checklist, which should be submitted along with the paper, is available for authors in
	preparing a manuscript employing qPCR.
</p>

<p>
  <a href="https://pubmed.ncbi.nlm.nih.gov/19246619/" target="_blank"><img src="./images/page/pdf_icon.png" alt="icon"/>&nbsp;&nbsp;&nbsp;Download the MIQE paper</a><br /><br />
  <a href="./files/docs/MIQE_checklist.xls" target="_blank"><img src="./images/page/xls_icon.png" alt="icon"/>&nbsp;&nbsp;&nbsp;Download the MIQE checklist</a><br /><br />
  <a href="./files/docs/MIQE_checklist.pdf" target="_blank"><img src="./images/page/pdf_icon.png" alt="icon"/>&nbsp;&nbsp;&nbsp;Download the MIQE checklist</a><br /><br />
  <a href="./files/docs/20090325_Letter-to-Editors.pdf" target="_blank"><img src="./images/page/pdf_icon.png"  alt="icon"/>&nbsp;&nbsp;&nbsp;Letter to Editors inviting them to adopt the MIQE standard for their papers </a><br /><br />
</p>

<p><span style="font-weight:bold;">Checklist for Quantitative PCR Assays</span></p>

<table class="table-striped">
  <tr><th>Item to check</th><th class="rdml-center-text">Importance</th></tr>
  <tr><td colspan="2" class="table-dark">Experimental design</td></tr>
  <tr><td>Definition of experimental and control groups</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Number within each group</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Assay carried out by core lab or investigator's lab?</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Acknowledgement of authors' contributions</td><td class="rdml-center-text">D</td></tr>
  <tr><td colspan="2" class="table-dark">Sample</td>
  <tr><td>Description</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Volume/mass of sample processed</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Microdissection or macrodissection</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Processing procedure</td><td class="rdml-center-text">E</td></tr>
  <tr><td>If frozen - how and how quickly?</td><td class="rdml-center-text">E</td></tr>
  <tr><td>If fixed - with what, how quickly?</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Sample storage conditions and duration (especially for FFPE samples)</td><td class="rdml-center-text">E</td></tr>
  <tr><td colspan="2" class="table-dark">Nucleic acid extraction</td>
  <tr><td>Procedure and/or instrumentation</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Name of kit and details of any modifications</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Source of additional reagents used</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Details of DNase or RNase treatment</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Contamination assessment (DNA or RNA)</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Nucleic acid quantification</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Instrument and method</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Purity (A260/A280)</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Yield</td><td class="rdml-center-text">D</td></tr>
  <tr><td>RNA integrity method/instrument</td><td class="rdml-center-text">E</td></tr>
  <tr><td>RIN/RQI or Cq of 3' and 5' transcripts</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Electrophoresis traces</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Inhibition testing (Cq dilutions, spike or other)</td><td class="rdml-center-text">E</td></tr>
  <tr><td colspan="2" class="table-dark">Reverse transcription</td>
  <tr><td>Complete reaction conditions</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Amount of RNA and reaction volume</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Priming oligonucleotide (if using GSP) and concentration</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Reverse transcriptase and concentration</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Temperature and time</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Manufacturer of reagents and catalogue numbers</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Cqs with and without RT</td><td class="rdml-center-text">D*</td></tr>
  <tr><td>Storage conditions of cDNA</td><td class="rdml-center-text">D</td></tr>
  <tr><td colspan="2" class="table-dark">qPCR target information</td>
  <tr><td>If multiplex, efficiency and LOD of each assay</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Sequence accession number</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Location of amplicon</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Amplicon length</td><td class="rdml-center-text">E</td></tr>
  <tr><td>In silico specificity screen (BLAST, etc)</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Pseudogenes, retropseudogenes or other homologs?</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Sequence alignment</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Secondary structure analysis of amplicon</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Location of each primer by exon or intron (if applicable)</td><td class="rdml-center-text">E</td></tr>
  <tr><td>What splice variants are targeted?</td><td class="rdml-center-text">E</td></tr>
  <tr><td colspan="2" class="table-dark">qPCR oligonucleotides</td>
  <tr><td>Primer sequences</td><td class="rdml-center-text">E</td></tr>
  <tr><td>RTPrimerDB identification number</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Probe sequences</td><td class="rdml-center-text">D**</td></tr>
  <tr><td>Location and identity of any modifications</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Manufacturer of oligonucleotides</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Purification method</td><td class="rdml-center-text">D</td></tr>
  <tr><td colspan="2" class="table-dark">qPCR protocol</td>
  <tr><td>Complete reaction conditions</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Reaction volume and amount of cDNA/DNA</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Primer, (probe), Mg++ and dNTP concentrations</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Polymerase identity and concentration</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Buffer/kit identity and manufacturer</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Exact chemical constitution of the buffer</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Additives (SYBR Green I, DMSO, etc.)</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Manufacturer of plates/tubes and catalog number</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Complete thermocycling parameters</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Reaction setup (manual/robotic)</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Manufacturer of qPCR instrument</td><td class="rdml-center-text">E</td></tr>
  <tr><td colspan="2" class="table-dark">qPCR validation</td>
  <tr><td>Evidence of optimisation (from gradients)</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Specificity (gel, sequence, melt, or digest)</td><td class="rdml-center-text">E</td></tr>
  <tr><td>For SYBR Green I, Cq of the NCT</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Standard curves with slope and y-intercept</td><td class="rdml-center-text">E</td></tr>
  <tr><td>PCR efficiency calculated from slope</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Confidence interval for PCR efficiency or standard error</td><td class="rdml-center-text">D</td></tr>
  <tr><td>r2 of standard curve</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Linear dynamic range</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Cq variation at lower limit</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Confidence intervals throughout range</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Evidence for limit of detection</td><td class="rdml-center-text">E</td></tr>
  <tr><td>If multiplex, efficiency and LOD of each assay</td><td class="rdml-center-text">E</td></tr>
  <tr><td colspan="2" class="table-dark">Data analysis</td>
  <tr><td>qPCR analysis program (source, version)</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Cq method determination</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Outlier identification and disposition</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Results of NTCs</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Justification of number and choice of reference genes</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Description of normalisation method</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Number and concordance of biological replicates</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Number and stage (RT or qPCR) of technical replicates</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Repeatability (intra-assay variation)</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Reproducibility (inter-assay variation, %CV)</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Power analysis</td><td class="rdml-center-text">D</td></tr>
  <tr><td>Statistical methods for result significance</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Software (source, version)</td><td class="rdml-center-text">E</td></tr>
  <tr><td>Cq or raw data submission using RDML</td><td class="rdml-center-text">D</td></tr>
</table>

<p>
  <br />
</p>
<p>MIQE checklist for authors, reviewers and editors. All essential information (E) must be submitted with the
  manuscript.  Desirable information (D) should be submitted if available. If using primers obtained from RTPrimerDB,
  information on qPCR target, oligonucleotides, protocols and validation is available from that source.
</p>
<p><strong>*</strong>: Assessing the absence of DNA using a no RT assay is essential when first extracting RNA. Once
  the sample has been validated as RDNA-free, inclusion of a no-RT control is desirable, but no longer essential.
</p>
<p><strong>**</strong>: Disclosure of the probe sequence is highly desirable and strongly encouraged. However, since
  not all commercial pre-designed assay vendors provide this information, it cannot be an essential requirement. Use
  of such assays is advised against.
</p>

